channels/ro.m3u

```diff
@@ -51,10 +55,12 @@ http://cdn.credonet.tv:1935/ctv/smil:livecredo.smil/playlist.m3u8
 http://telekomtv-ro.akamaized.net/shls/LIVE$CredoTV/6.m3u8/Level(1677721)?start=LIVE&end=END
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="https://i.imgur.com/XzChc5P.jpg" group-title="",Cromtel TV
 http://webmobile.xdev.ro:81/tv7/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="" group-title="",Diaspora TV
+http://diasporamedia.go.ro:1935/diasporatv/live/playlist.m3u8
 #EXTINF:-1 tvg-id="Digi24 RO" tvg-name="Digi24 RO" tvg-language="Romanian" tvg-logo="https://i.imgur.com/MufRJgP.png" group-title="News",Digi24
 http://81.196.0.126/digi24edge/digi24hdhqhls/index.m3u8
 #EXTINF:-1 tvg-id="Digi24 RO" tvg-name="Digi24 RO" tvg-language="Romanian" tvg-logo="https://i.imgur.com/MufRJgP.png" group-title="News",Digi24
-http://81.196.0.126:80/digi24edge/digi24hdhqhls/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="" group-title="", TVPLUS SVhttp://81.196.0.126:80/digi24edge/digi24hdhqhls/index.m3u8
```

Comment:
- link should be on a separate line
- there's no need to put a space before channel name
- the link leads to the broadcast of Digi24 although in the name TVPLUS